lib/spack/llnl/util/cpu/microarchitectures.json

```diff
@@ -694,6 +694,39 @@
         }
       }
     },
+    "k10": {
+      "from": "x86_64",
+      "vendor": "AuthenticAMD",
+      "features": [
+        "mmx",
+        "sse",
+        "sse2",
+        "sse4a",
+        "abm",
+        "cx16",
+        "popcnt",
+        "3dnow",
+        "3dnowext"
+      ],
+      "compilers": {
+        "gcc": {
+          "name": "amdfam10",
+          "versions": "4.2:",
```

Comment:
This should be `4.3:`. Thanks for catching that @alalazo

Reply:
For your convenience, the reference for [GCC 4.2](https://gcc.gnu.org/onlinedocs/gcc-4.2.4/gcc/i386-and-x86_002d64-Options.html#i386-and-x86_002d64-Options) and for [GCC 4.3](https://gcc.gnu.org/onlinedocs/gcc-4.3.6/gcc/i386-and-x86_002d64-Options.html#i386-and-x86_002d64-Options).

Reply:
Yeah, I'd done the checks but apparently fat-fingered the coding.